JS/Lesson22/goksel/script.js

```diff
@@ -0,0 +1,24 @@
+let contents = document.querySelectorAll(".content");
+
+let buttons = document.querySelectorAll(".click-btn");
+
+buttons.forEach((btn,index)  => {
+    btn.addEventListener("click",function(){
+        contents[index].classList.toggle("active");
+        btn.classList.toggle("active");
+
+
+        if(contents[index].classList.contains("active")){
```

Comment:
Since you add a remove class active already based on the click event, you only need to add styles in CSS. So you don't need this entire part of code (if-else statement).